### PR TITLE
DBus tests for bcache module

### PR DESCRIPTION
**Pull request by vojtechtrefny**
First commit (_Fix PolicyKit..._) is from #162

### CODE REVIEW

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

File: src/tests/dbus-tests/test_bcache.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        cls.ensure_modules_loaded()
+
+    def _wipe(self, device):
+        for _ in range(10):
+            ret, _out = self.run_command('wipefs -a %s' % device)
+            if ret == 0:
+                break
+            time.sleep(1)
+        else:
+            self.fail('Failed to wipe device %s after removing bcache.' % device)
```
Comment (vpodzime) [inline src/tests/dbus-tests/test_bcache.py]: This looks like a useful generic function for the ``StoragedTestCase`` itself.

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

File: src/tests/dbus-tests/test_bcache.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        # cache_mode contains all cache modes, the right one is in square brackets
+        modes = self.read_file('/sys/block/%s/bcache/cache_mode' % bcache_name)
+        return next((x[1:-1] for x in modes.split() if re.match(r'^\[.*\]$', x)), None)
+
+    def _get_size(self, bcache_name):
+        cache_dir = '/sys/block/%s/bcache/cache' % bcache_name
+
+        # sum sizes from all caches
+        caches = ['%s/%s' % (cache_dir, d) for d in os.listdir(cache_dir) if re.match('cache[0-9]*$', d)]
+        size = sum([int(self.read_file(os.path.realpath(c) + '/../size')) for c in caches])
```
Comment (vpodzime) [inline src/tests/dbus-tests/test_bcache.py]: Extra square brackets here.

File: src/tests/dbus-tests/test_bcache.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+        return size
+
+    def test_create_destroy(self):
+        '''Test creating a new bcache and its properties'''
+
```
Comment (vpodzime) [inline src/tests/dbus-tests/test_bcache.py]: Please make sure the modules are loaded here (``StoragedTestCase.ensure_modules_loaded``)

File: src/tests/dbus-tests/test_bcache.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        # make sure the bcache device is not on dbus
+        udisks = self.get_object('')
+        objects = udisks.GetManagedObjects(dbus_interface='org.freedesktop.DBus.ObjectManager')
+        self.assertNotIn(str(bcache.object_path), objects.keys())
+
+        self.assertFalse(os.path.exists('/dev/%s' % bcache_name))
+
+    def test_set_mode(self):
+        '''Test if it's possible to change cache mode on existing bcache'''
+
```
Comment (vpodzime) [inline src/tests/dbus-tests/test_bcache.py]: And the same here or rename the tests to make sure this one runs after the create_destroy one.

File: src/tests/dbus-tests/test_bcache.py (hunk 2 above)
Comment (vojtechtrefny) [inline src/tests/dbus-tests/test_bcache.py]: Fixed.

File: src/tests/dbus-tests/test_bcache.py (hunk 1 above)
Comment (vojtechtrefny) [inline src/tests/dbus-tests/test_bcache.py]: Fixed.

File: src/tests/dbus-tests/test_bcache.py (hunk 3 above)
Comment (vojtechtrefny) [inline src/tests/dbus-tests/test_bcache.py]: I call `ensure_modules_loaded` in `setUpClass`.

**Pull request merged**